tool/tctl/common/plugin/plugins_command.go

```diff
@@ -330,6 +332,8 @@ func (p *PluginsCommand) TryRun(ctx context.Context, cmd string, clientFunc comm
 		commandFunc = p.InstallEntra
 	case p.install.netIQ.cmd.FullCommand():
 		commandFunc = p.InstallNetIQ
+	case p.install.awsIC.cmd.FullCommand():
+		commandFunc = p.InstallAWSIC
```

Comment:
Does plugin deletion works by default? If yes, i think we should support resource cleanup or guide users to do it manually before letting them delete the plugin.